proxy_server.py

```diff
@@ -59,12 +57,14 @@ class AddProxyServerHandler(webapp2.RequestHandler):
 
   """Handler for adding new proxy servers."""
 
-  @admin.RequireAdmin
+  @admin.OAUTH_DECORATOR.oauth_required
+  @admin.RequireAppAndDomainAdmin
```

Comment:
Cool, this looks much cleaner than before!  

Just one small question: does the ordering of these decorators matter? If the decorators are called from the bottom up, and RequireAppAndDomainAdmin uses OAUTH_DECORATOR, then it's okay to call @admin.OAUTH_DECORATOR.oauth_required afterwards?